setversion.sh

```diff
@@ -48,7 +48,7 @@ update_plugin_xml ()
     local version=$2
     local isDev=$3
     local newPodSpecVersion="tag=\"v${version}\""
-    gsed -i "s/\($[ ]\)version.*=.*\"[^\"]*\">/\1version=\"${version}\">/g" ${file}
+    gsed -i "s/\($[ ]*\)version.*=.*\"[^\"]*\">/\1version=\"${version}\">/g" ${file}
```

Comment:
Because of that, the tagged v9.2.1 version of plugin.xml still says v9.2.0. Should I fix master and move the tag?

Reply:
(I just fixed master and moved the tag)